docs/ref/templates/builtins.txt

```diff
@@ -1628,6 +1628,34 @@ For example::
 If ``value`` is ``"testing\r\njavascript \'string" <b>escaping</b>"``,
 the output will be ``"testing\\u000D\\u000Ajavascript \\u0027string\\u0022 \\u003Cb\\u003Eescaping\\u003C/b\\u003E"``.
 
+.. templatefilter:: json_tag
+
+``json_tag``
+-----------
+
+.. versionadded:: 2.1
+
+Outputs an object JSON-encoded, wrapped in a <script> tag ready for use with JavaScript.
+
+**Argument:** HTML "id" of the <script> tag.
+
+For example::
+
+    {{ value|json_tag:"value_json" }}
```

Comment:
Perhaps a better id than `value_json` would make the example clearer, e.g. `hello-data`?